problem_3/lib/largest_prime_factor.rb

```diff
@@ -0,0 +1,24 @@
+#What is the largest prime factor of the number 600851475143 ?
+
+class PrimeFactor
+
+	def initialize(limit)
+		@limit = limit
+	end
+
+	def factor_finder
+		div = 2
+		largest_factor = 0
+		while (div <= @limit)
```

Comment:
Optional: This can be can optimized by checking up to the square root of `@limit`